analysis/runtime/src/mir_loc.rs

```diff
@@ -169,6 +169,8 @@ pub struct EventMetadata {
     pub destination: Option<MirPlace>,
     /// Destination func [`DefPathHash`] of [`Event`](crate::events::Event).
     pub transfer_kind: TransferKind,
+    /// Any string useful for debugging
```

Comment:
```suggestion
    /// Any string useful for debugging.
```

Reply:
Fixed in https://github.com/immunant/c2rust/pull/566/commits/0e2477d482a5b9f58a6b904f66243d6070d1af24.